### ISSUE TITLE
No sample data or instruction

### ISSUE THREAD

**Issue by stupidfive** [gharriso/MongoDBPerformanceTuningBook#1]
The readme file says

> The [sampleData](https://github.com/gharriso/MongoDBPerformanceTuningBook/blob/master/sampleData) directory contains all the data used in our examples as a compressed dump file. Instructions on how to load the data can be found in the same folder.

But the `sampleData` directory looks like this

```
.
├── customerDOBFix.js
├── customersDateOfBirth.js
├── growCustomers.js
└── modCustomers.js
```

There is no dump file or instruction.

**Comment by pandulaDW** [comment]
> 

Same here

**Comment by pandulaDW** [comment]
> The readme file says
> 
> > The [sampleData](https://github.com/gharriso/MongoDBPerformanceTuningBook/blob/master/sampleData) directory contains all the data used in our examples as a compressed dump file. Instructions on how to load the data can be found in the same folder.
> 
> But the `sampleData` directory looks like this
> 
> ```
> .
> ├── customerDOBFix.js
> ├── customersDateOfBirth.js
> ├── growCustomers.js
> └── modCustomers.js
> ```
> 
> There is no dump file or instruction.

Same here

**Comment by Riding-A-Colt** [comment]
Yes, I meet the same problem. When I read chapter two, I try to find the 'customer' db, but there is nothing.
